cli/pkg/cmd/create/helm.go

```diff
@@ -75,7 +75,11 @@ func (h *Helm) Setup() error {
 }
 
 func (h *Helm) InstallCharts(overrides []string) error {
-	return h.runHelm(append([]string{"install", "--values", "/apps/config/airy-config-map.yaml", "core", "/apps/helm-chart/"}, overrides...))
+	return h.runHelm(append([]string{"install",
+		"--values", "/apps/config/airy-config-map.yaml",
+		"--set", "global.appImageTag=" + h.version,
+		"--set", "global.namespace=" + h.namespace,
```

Comment:
The cli namespace was also not being used.

Reply:
this-is-also-fine.gif